pages/product.html

```diff
@@ -18,44 +18,63 @@
       >
```

Comment:
![Screenshot (68)](https://user-images.githubusercontent.com/72399705/154805299-c6f82341-1617-4b08-ab42-bcc8c600f00b.png)
Your filter sidebar is overlapping your footer. Please look into it.

Reply:
Fixed it. Thanks